src/names/preludes.md

```diff
@@ -130,13 +141,19 @@ This attribute does not affect the [language prelude].
 [`alloc`]: ../../alloc/index.html
 [`Box`]: ../../std/boxed/struct.Box.html
 [`core::prelude::v1`]: ../../core/prelude/index.html
+[`core::prelude::rust_2015`]: ../../core/prelude/index.html
+[`core::prelude::rust_2018`]: ../../core/prelude/index.html
+[`core::prelude::rust_2021`]: ../../core/prelude/index.html
```

Comment:
Is there a particular reason not to link the to the specific modules?  For example, [`rust_2021`](https://doc.rust-lang.org/core/prelude/rust_2021/index.html) precisely shows what is exported from the module.

I realize the original `v1` links went to the top-level module, but I honestly don't remember why I did that (I think linking to [`v1`](https://doc.rust-lang.org/1.25.0/std/prelude/v1/index.html) would also be more clear).

Reply:
I've changed them, including v1